pages/profile/index.js

```diff
@@ -21,7 +21,10 @@ module.exports = () => {
     }
     return req.api(`/establishment/${req.establishment}/profile/${profile}`)
       .then(({ json: { data, meta } }) => {
-        req.model = cleanModel(data);
+        const model = cleanModel(data);
+        model.exemptions = model.trainingModules.filter(m => { return m.exemption; });
+        model.trainingModules = model.trainingModules.filter(m => { return !m.exemption; });
```

Comment:
At least one of the functional tests is failing because `model.trainingModules` is not defined here.

It is probably advisable to ensure that the `trainingModules` falls back to an empty array before trying to filter it.